openscad.pro

```diff
@@ -512,7 +513,8 @@ SOURCES += \
            src/input/InputEventMapper.cc \
            src/input/InputDriverManager.cc \
            src/input/AxisConfigWidget.cc \
-           src/input/ButtonConfigWidget.cc
+           src/input/ButtonConfigWidget.cc \
+           src/input/WheelIgnorer.cc
```

Comment:
At some point, adding the inputdriver stuff to the top-level `CMakeLists.txt` would be nice. May be out of scope for this particular PR though.